### PR TITLE
Add hotkeys blocklist and turn off hotkey seeking when any live

### LINKED ISSUE #336: Bug: user can seek video with keyboard in live mode
### Checklist

- [x] Verified whether your issue exists in the latest version Mux Element(s) release: `npm install @mux/[element_name]` or `yarn @mux/[element_name]`.
  - (NOTE: If the issue is not occurring in the latest version, we may not resolve it.)
- [x] Confirmed that this issue hasn't [already been filed](https://github.com/muxinc/elements/issues?q=is%3Aopen+is%3Aissue+label%3Abug).

### Which Mux Elements/Packages has a bug?

- common

### Which browser(s) are you using?

- Chrome

### Which operating system(s) are you using?

macOS

### How are you using Mux Elements?

npm package

### Describe the bug

When the player is setup in 'live' mode the controls to seek the video are correctly not shown. The user can still seek the video back using the arrow keys

## Steps To Reproduce

1. Set the player in 'live' mode
2. Use the arrow keys to seek the video back

## The expected behavior

In live, seeking the video should be forbidden

### PR DESCRIPTION
**Pull request by gkatsev**
Depends on https://github.com/muxinc/media-chrome/pull/286
Fixes #336

### CODE REVIEW

File: packages/mux-player/README.md [hunk 1]
```diff
@@ -161,6 +161,29 @@ Supported **tokens**:
...
+### hotkeys
+
+Offers a way to turn off keyboard shortcuts that should not be enabled.
+The `hotkeys` attribute accepts a blocklist as a space separated string.
+
+Supported **tokens**:
+`nof`, `nok`, `nom`, `noarrowleft`, `noarrowright`, `nospace`
+
+```html
+<mux-player playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe" controlslist="noarrowleft noarrowright"></mux-player>
```
Comment (luwes) [inline packages/mux-player/README.md:173]:
```suggestion
<mux-player playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe" hotkeys="noarrowleft noarrowright"></mux-player>
```

Comment (gkatsev) [reply packages/mux-player/README.md:173]: haha, oops. Copypasta error.

File: packages/mux-player/src/index.ts [hunk 2]
```diff
@@ -621,6 +627,10 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.AUDIO, '');
   }
 
+  get hotKeys() {
+    return this.#hotKeys;
+  }
+
   get nohotkeys() {
```
Comment (luwes) [inline packages/mux-player/src/index.ts:634]: hmm I feel like we're not following the convention here. should be 

```suggestion
  get noHotKeys() {
```

maybe in the props map access `this.hasAttribute(PlayerAttributes.NOHOTKEYS)` directly

**Comment by gkatsev** [comment]
This also needs an update to mux-player-react to include the prop in the types.

File: packages/mux-player/src/index.ts (hunk 2 above)
Comment (cjpillsbury) [reply packages/mux-player/src/index.ts:634]: if the attribute name is `nohotkeys`, the property name should also be `nohotkeys` (based on our translation, which assumes hyphens in attributes denote camel case capitalization)

Comment (gkatsev) [reply packages/mux-player/src/index.ts:638]: does that mean I should rename `get hotKeys` to `get hotkeys`?

Comment (gkatsev) [reply packages/mux-player/src/index.ts:638]: And should it be renamed in media-chrome too? (I haven't merged that PR yet)

File: packages/mux-player/src/types.d.ts [hunk 3]
```diff
@@ -33,6 +34,7 @@ export type MuxTemplateProps = Partial<MuxPlayerProps> & {
   metadataVideoTitle: string;
   metadataViewerUserId: string;
   noHotKeys: boolean;
+  hotKeys: AttributeTokenList;
```
Comment (cjpillsbury) [inline packages/mux-player/src/types.d.ts:37]: issue(blocker): This is inaccurate.

File: packages/mux-player/src/index.ts [hunk 4]
```diff
@@ -616,6 +622,10 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.AUDIO, '');
   }
 
+  get hotkeys() {
+    return this.#hotKeys;
```
Comment (cjpillsbury) [inline packages/mux-player/src/index.ts:626]: question(blocking): is there a reason we're naming the private prop camel case but the public prop lowercase? I have a feeling this is confusing/will confuse things.

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
a few callouts.

File: packages/mux-player/src/types.d.ts (hunk 3 above)
Comment (gkatsev) [reply packages/mux-player/src/types.d.ts:37]: oops, forgot to update this one. Good callout.

File: packages/mux-player/package.json [hunk 5]
```diff
@@ -52,7 +52,7 @@
     "@github/template-parts": "^0.5.3",
     "@mux/mux-video": "0.8.4",
     "@mux/playback-core": "0.9.0",
-    "media-chrome": "0.9.0"
+    "media-chrome": "0.9.1-canary.2-eb0d770"
```
[the remark this answers is not in the document]
Comment (gkatsev) [reply packages/mux-player/package.json:55]: Yup, this PR won't get merged with a canary build. It's there for testing. I'll either remove it or get a real media-chrome release updated in here before merging.

File: packages/mux-player/src/types.d.ts (hunk 3 above)
Comment (cjpillsbury) [reply packages/mux-player/src/types.d.ts:38]: cool that tracks. Resolving.

**Review by cjpillsbury — APPROVED** [review]
pre-approving on the assumption that this will not be merged until after Media Chrome release + dependency update.

**Pull request merged**